pkg/proto/nbcontract/v1/gpuconfig.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package nbcontract.v1;
+
+import "pkg/proto/nbcontract/v1/enum.proto";
+
+message GpuConfig {
+  // nvidia_state represents the Nvidia GPU state
+  FeatureState status = 1;
+
+  // FeatureState is_mig_node = 2; removed since it only depends on gpu_instance_profile
```

Comment:
@anujmaheshwari1 I think we can keep the removed variables with comments for now. That will be easier to track what are removed and why it could be removed. Then we can have a clean-up at the end of development stage. Thoughts?